utils/base_expiration_command.py

```diff
@@ -27,6 +27,11 @@ def add_arguments(self, parser):
             action="store_true",
             help=f"Run the command even if {self.feature_flag_name} is not set",
         )
+        parser.add_argument(
```

Comment:
So this new argument will not yet be used anywhere e.g. in the cronjobs?

Reply:
nope. Not used yet, but can be used if needed. I just wanted to add it there already, but didn't want to play with the cron jobs, so set the default to be "excluded".